sunpy/net/tests/test_helioviewer.py

```diff
@@ -35,44 +35,24 @@ class TestHelioviewerClient:
     def test_get_datasources(self, client):
         """Makes sure datasource query returns a valid result and source id
         is casted to an integer"""
-        assert type(client.sources['SDO']['AIA']['AIA']['171']['sourceId']) is int
-
-    def test_get_closest_image(self, client):
-        """Tests getClosestImage API method"""
-        # check basic query
-        im1 = client.get_closest_image('1994/01/01',
-                                       observatory='SOHO',
-                                       instrument='EIT',
-                                       detector='EIT',
-                                       measurement='195')
-        assert im1['width'] == im1['height'] == 1024
-
-        # result should be same when using source id to query
-        source_id = client.sources['SOHO']['EIT']['EIT']['195']['sourceId']
-
-        im2 = client.get_closest_image('1994/01/01', sourceId=source_id)
-
-        assert im1 == im2
```

Comment:
Was there a reason to delete all this and not just rewrite the functions?

Reply:
I deleted it because this function just used sourceId to do the testing but now sourceId is a compulsory part of the function call so I guess changing it won't be that benificial

Reply:
Yeah I see. Hmm. I wonder if it can be used to check something else.